kong/db/strategies/connector.lua

```diff
@@ -5,8 +5,8 @@ local fmt = string.format
 local Connector = {
   defaults = {
     pagination = {
-      page_size     = 1000,
-      max_page_size = 50000,
+      page_size     = 512,  -- work with lmdb
+      max_page_size = 512,  -- work with lmdb
```

Comment:
Can we have different default for LMDB?

Reply:
This should be changed after https://github.com/Kong/lua-resty-lmdb/pull/58/files is merged. Currently setting it to other value (specifically, one that is different from [`DEFAULT_OPS_SIZE`](https://github.com/Kong/lua-resty-lmdb/blob/5016b110dfc0e0a7a2742c63a6c922d5c7d75218/lib/resty/lmdb/prefix.lua#L14) causes issues) for DAO's more page detection :(

Reply:
KAG-5565